src/widget/sidebar.c

```diff
@@ -648,21 +454,21 @@ static void draw_sliding_foreground(void)
     draw_overlay_text(x_offset_expanded + 4);
     draw_build_image(x_offset_expanded + 6, 1);
 
-    draw_sidebar_filler(x_offset_collapsed, FILLER_Y_OFFSET, 1);
+    sidebar_filler_draw_sidebar_filler(x_offset_collapsed, FILLER_Y_OFFSET, 1);
 
     draw_sidebar_remainder(x_offset_expanded, 0);
-    draw_extra_info_buttons(x_offset_expanded, 0);
+    sidebar_filler_draw_extra_info_buttons(x_offset_expanded, 0);
```

Comment:
hmmm, draw_extra_info_buttons for sure, but draw_sidebar_remainder would have to happen with or without sidebar_filler stuff happening, since the sidebar always goes to the bottom of the screen.  

can i leave that one in sidebar.c?

thinking about it more... i wonder if all calls to sidebar_filler functions in sidebar.c should be done from within draw_sidebar_remainder in sidebar.c, since the filler is inserted inside the remainder area.  gonna try this out, let me know what you think!

Reply:
ohh its already mostly like that, just the slide function was doing it differently.  throwing another arg to draw_remainder to indicate a slide is occurring should clean this up.